Evaluate-ModelarDB-Compression/main.py

```diff
@@ -0,0 +1,281 @@
+""" Script for evaluating ModelarDB with different data sets. """
```

Comment:
This comment should not have a leading and trailing space. It also does not seem like the other utility files have a module doc comment so this could also be removed.

Reply:
The formatting was copied from [Evaluate-ModelarDB-Changes/main.py](https://github.com/ModelarData/Utilities/blob/main/Evaluate-ModelarDB-Changes/main.py) so I have removed the spaces in both `main.py` files.